mpf/devices/ball_hold.py

```diff
@@ -68,17 +68,12 @@ def _initialize(self):
 
         self.source_playfield = self.config['source_playfield']
 
-    @event_handler(10)
-    def enable(self, **kwargs):
+    def enable(self):
         """Enable the hold.
 
         If the hold is not enabled, no balls will be held.
-
-        Args:
-            **kwargs: unused
         """
-        del kwargs
-
+        super().enable()
```

Comment:
I'm wondering about the choice to override `self.enable()` here and call super, instead of using `EnableDisableMixin` and overriding `self._enable()`.

Reply:
Good idea. I will review that. Maybe we should add disable to the base class also (but that would something for another PR).

Reply:
done